python/python/lancedb/query.py

```
@@ -106,193 +128,187 @@ def query_type(self) -> FullTextQueryType:
         str
             The type of the query.
         """
+        return FullTextQueryType(self._inner.query_type())
+
+    def model_post_init(self, __context):
+        self._inner = self.build_inner()
+
+    def build_inner(self) -> PyFullTextQuery:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)